control/autoware_autonomous_emergency_braking/README.md

```diff
@@ -0,0 +1,225 @@
+
+![rss_check](./image/rss_check.drawio.svg)
+
+### 6. Send emergency stop signals to `/diagnostics`
+
+If AEB detects collision with point cloud obstacles in the previous step, it sends emergency signal to `/diagnostics` in this step. Note that in order to enable emergency stop, it has to send ERROR level emergency. Moreover, AEB user should modify the setting file to keep the emergency level, otherwise Autoware does not hold the emergency state.
+
+## Use cases
+
+### Front vehicle suddenly brakes
+
+The AEB can activate when a vehicle in front suddenly brakes, and a collision is detected by the AEB module. Provided the distance between the ego vehicle and the front vehicle is large enough and the ego’s emergency acceleration value is high enough, it is possible to avoid or soften collisions with vehicles in front that suddenly brake. NOTE: the acceleration used by the AEB to calculate rss_distance is NOT necessarily the acceleration used by the ego while doing an emergency brake. The acceleration used by the real vehicle can be tuned by changing the [mrm_emergency stop jerk and acceleration values](https://github.com/tier4/autoware_launch/blob/d1b2688f2788acab95bb9995d72efd7182e9006a/autoware_launch/config/system/mrm_emergency_stop_operator/mrm_emergency_stop_operator.param.yaml#L4).
+
+![front vehicle collision prevention](./image/front_vehicle_collision.drawio.svg)
+
+### Stop for objects that appear suddenly
+
+When an object appears suddenly, the AEB can act as a fail-safe to stop the ego vehicle when other modules fail to detect the object on time. If sudden object cut ins are expected, it might be useful for the AEB module to detect collisions of objects BEFORE they enter the real ego vehicle path by increasing the `expand_width` parameter.
+
+![occluded object collision prevention](./image/occluded_space.drawio.svg)
+
+### Preventing Collisions with rear objects
+
+The AEB module can also prevent collisions when the ego vehicle is moving backwards.
+
+![backward driving](./image/backward-driving.drawio.svg)
+
+### Preventing collisions in case of wrong Odometry (IMU path only)
+
+When vehicle odometry information is faulty, it is possible that the MPC fails to predict a correct path for the ego vehicle. If the MPC predicted path is wrong, collision avoidance will not work as intended on the planning modules. However, the AEB’s IMU path does not depend on the MPC and could be able to predict a collision when the other modules cannot. As an example you can see a figure of a hypothetical case in which the MPC path is wrong and only the AEB’s IMU path detects a collision.
+
+![backward driving](./image/wrong-mpc.drawio.svg)
+
+## Parameters
```

Comment:
Please also add the following parameters to the list.

- cluster_tolerance
- cluster_minimum_height
- minimum_cluster_size
- maximum_cluster_size

Reply:
@sfukuta apparently I cannot push new commits to this branch since I get this error: 
`remote: error: GH006: Protected branch update failed for refs/heads/beta/v0.3.18.2+AEB-update.
remote: 
remote: - Changes must be made through a pull request.
remote: 
remote: - 2 of 2 required status checks are expected.
To https://github.com/tier4/autoware.universe.git
 ! [remote rejected]     beta/v0.3.18.2+AEB-update -> beta/v0.3.18.2+AEB-update (protected branch hook declined)
error: failed to push some refs to 'https://github.com/tier4/autoware.universe.git'`

if your only concerns are about the README please approve this PR and I will make a separate PR to fix the README